includes/Core/Tags/Enhanced_Conversions/Enhanced_Conversions.php

```diff
@@ -67,6 +79,24 @@ protected function get_user_data() {
 		return $user_data;
 	}
 
+	/**
+	 * Checks if any module using GTag is connected.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return boolean Whether any module using GTag is connected.
+	 */
+	protected function has_connected_gtag_modules() {
```

Comment:
I'd suggest renaming this to be singular, we only need to know one module is connected as per my other comment, and there may indeed only be one connected module.

```suggestion
	protected function has_connected_gtag_module() {
```